tvb_framework/tvb/interfaces/web/templates/jinja2/project/editone.html

```diff
@@ -82,7 +82,28 @@ <h4>
                     <dd>
                         <p class="field-data"><input tabindex='3' id="administrator" type="text"
                                                      value="{{ data.administrator if ('administrator' in data) else '' }}"
-                                                     disabled="disabled"/></p>
+                                                     disabled="disabled"/>
+                        </p>
+                    </dd>
+
+                    <dt>
+                        <label for="disable_imports">Disable imports for this project</label>
+                    </dt>
+
+                    <dd>
+                        <p class="field-data"><input tabindex='4' id="disable_imports" type="checkbox"
+                                                     name="disable_imports" {{ 'checked' if data.disable_imports }} {{ 'disabled' if not editUsersEnabled }}/>
+                        </p>
+                    </dd>
+
+                    <dt>
+                        <label for="max_operation_size">Max allowed size for one operation (KB)</label>
+                    </dt>
+
+                    <dd>
+                        <p class="field-data"><input tabindex='5' id="max_operation_size" type="number" name="max_operation_size"
+                                                     value="{{ data.max_operation_size }}" {{ 'disabled' if not editUsersEnabled }}/>
+                        </p>
```

Comment:
I believe after these, somewhere on the page `tabindex='4'` exists already, and it should be change into `tabindex='6'`

And I believe that `max_operation_size ` could be a validated field (e.g. to be a positive number), thus an error message would be possible to appear nearby.